rust/connlib/libs/common/src/control.rs

```diff
@@ -339,8 +356,10 @@ enum OkReply<T> {
 #[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq)]
 #[serde(rename_all = "snake_case")]
 pub enum ErrorInfo {
-    Reason(String),
```

Comment:
@conectado: Minor bug fix here. I imagine you intended this to be a catch-all variant, right? That doesn't work unfortunately because `serde` can't just fit everything into a string. In order to not fail deserialization, we have to ditch the body and mark an empty variant as `serde(other)`.

Reply:
I wasn't indending this to be a catch-all, sometimes the phoenix channel returns an error:

```
status: error, response: { reason: <reason string> }
```

Reply:
I should add more test cases

Reply:
Ah right, I'll revert that then!